From b0df678efa12ea3f74e5c989b0bd2a27f41520c5 Mon Sep 17 00:00:00 2001
From: Alexandre Frade <kernel@xanmod.org>
Date: Fri, 30 May 2025 19:58:58 +0000
Subject: [PATCH 14/18] XANMOD: mm/vmscan: Reduce amount of swapping

Signed-off-by: Alexandre Frade <kernel@xanmod.org>
---
 mm/vmscan.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index 614ccf39fe3f..057e146441fa 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -199,7 +199,7 @@ struct scan_control {
 /*
  * From 0 .. MAX_SWAPPINESS.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 10;
 
 #ifdef CONFIG_MEMCG
 
-- 
2.47.2

